### PR TITLE
fix: extend @types/elasticsearch IndicesUpdateAliasesParamsAction parameters

### PR DESCRIPTION
**Pull request by NicolasBiehler**
As shown in elasticsearch documentation (https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html#filtered), an alias can have a routing value and a filter specified.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html#filtered>>

### CODE REVIEW

**Comment by pushplay** [comment]
Looks good.  Can you add a couple tests for it?

File: types/elasticsearch/elasticsearch-tests.ts [hunk 1]
```diff
@@ -338,22 +338,42 @@ client.indices.updateAliases({
...
-        source: {
-          index: "twitter"
-        },
-        dest: {
-          index: "new_twitter"
+client.indices.updateAliases({
+  body: {
+    actions: [
+      {
+        add: {
```
Comment (deerawan) [inline types/elasticsearch/elasticsearch-tests.ts]: would you mind adding test for `remove` as well since we also change that part?

Comment (NicolasBiehler) [reply types/elasticsearch/elasticsearch-tests.ts]: Actually I rechecked the doc https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html#deleting) and these parameters are only available for the `add` command.

So I removed the change to the `remove` one altogether.

File: types/elasticsearch/elasticsearch-tests.ts [hunk 2]
```diff
@@ -338,22 +338,42 @@ client.indices.updateAliases({
...
+          index: 'logstash-2018.11', alias: 'logstash-filtered', filter: {
+            query: {
+              exists: { field: 'id' }
+            }
+          }, routing: 'id'
```
Comment (pushplay) [inline types/elasticsearch/elasticsearch-tests.ts]: Should have a new line before `routing`.

**Comment by pushplay** [comment]
Great work.  Just one small formatting fix.

I think the autoformatting changes are good.  I don't love 2-space tabbing but that's what's happening so it should be consistent.

**Review by pushplay — APPROVED** [review]

**Pull request merged**